shared/actions/wallets.js

```diff
@@ -57,6 +57,35 @@ const loadPayments = (
     })
   )
 
+const loadDisplayCurrencies = (state: TypedState, action: WalletsGen.LoadDisplayCurrenciesPayload) =>
+  RPCTypes.localGetDisplayCurrenciesLocalRpcPromise().then(res =>
+    WalletsGen.createDisplayCurrenciesReceived({
+      currencies: (res || []).map(c => Constants.currenciesResultToCurrencies(c)),
+    })
+  )
+
+const loadDisplayCurrency = (state: TypedState, action: WalletsGen.LoadDisplayCurrencyPayload) =>
+  RPCTypes.localGetDisplayCurrencyLocalRpcPromise({accountID: action.payload.accountID,
+  }).then(res =>
+    WalletsGen.createDisplayCurrencyReceived({
+      accountID: action.payload.accountID,
+      currency: Constants.makeCurrencies(res),
+    })
+  )
+
+const changeDisplayCurrency = (state: TypedState, action: WalletsGen.ChangeDisplayCurrencyPayload) =>
+  RPCTypes.localChangeDisplayCurrencyLocalRpcPromise({
+    accountID: action.payload.accountID,
+    currency: action.payload.code, // called currency, though it is a code
+  }).then(res =>
+    WalletsGen.createLoadDisplayCurrency({accountID: action.payload.accountID})
```

Comment:
This is to retrieve the new setting from the service again and refresh the display. I take it this will be triggered in the future by gregor instead?